arch/risc-v/src/bl602/bl602_irq_dispatch.c

```diff
@@ -41,7 +41,7 @@
  * riscv_dispatch_irq
  ****************************************************************************/
 
-void *riscv_dispatch_irq(uintptr_t vector, uintptr_t *regs)
+void *riscv_dispatch_irq(uintptr_t vector, uintreg_t *regs)
```

Comment:
this patch should merge to previous one

Reply:
do you mean squash the last two? or move the`riscv_dispatch_irq()` ones to the `uintreg_t`(as they are all CI fixes) and keep the rest in last one (as they are sematical changes needed for qemu-rv64ilp32 to work)?

Reply:
yes, each patch should be self-contained and pass the ci without helping from a future patch.